src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -88,7 +88,7 @@
       "displayName": "Application ID",
       "type": "parameter",
       "datatype": "string",
-      "description": "The application identifier (e.g. com.mycompany.myapp), used by the Android, iOS, and Mac targets."
+      "description": "The application identifier (e.g. com.mycompany.myapp), used by the Android, iOS targets."
```

Comment:
@jeromelaban, @kazo0 I removed mac here as if I am not mistaken since the Mac Catalyst platform was removed, then there is no need for an App ID for Skia Desktop macOS, right ?

Reply:
No, we need to keep it the desktop target uses it

Reply:
Jerome is okay with the current version